README.md

```diff
@@ -103,6 +103,23 @@ To use the `SidekiqPublisher`, this can be replaced by including
 `SidekiqPublisher::Worker`. The usual `perform_async`, etc methods will be
 available on the class but jobs will be staged in the Postgres table.
 
+### Tying to a transaction
+Jobs that are set up to use `sidekiq_publisher` will stage new
+instances to postgres within the currently active transaction. To
+guarantee that your job is run when there's a change to the system
+of record, publish it during the transaction representing that
+change. Usually, that can be accomplished by publishing in one of
+the ActiveRecord callbacks that are called within-transaction (e.g.
+`after_save`):
```

Comment:
It may be more useful to explicitly call out which callbacks are outside the transaction. I.e. today we enqueue most jobs in `after_commit` so that is the main one to avoid if you want to take advantage of the `sidekiq_publisher`.